benchmarks/Fractions.Benchmarks/Fractions.Benchmarks.csproj

```diff
@@ -2,7 +2,7 @@
 
   <PropertyGroup>
     <OutputType>Exe</OutputType>
-    <TargetFrameworks>net481;net8.0</TargetFrameworks>
+    <TargetFrameworks>net9.0;net48</TargetFrameworks>
```

Comment:
I think there was thing in past about having the framework project first, but it seems to be fine now...

Reply:
Unfortunately, this change doesn't make sense to me - nothing changes in the compiler result.

Reply:
I'm not sure of all the things that change in regards to the whole process, but if we put the framework first it uses it for the "host" and outputs the benchmark results in a `net481` folder instead of the `net9.0` folder where I would expect them to be. I think this also affects the nuget packages it's using which cause the warning (the auto-generated projects seem to be fine as far as i can tell).